depthai_sdk/examples/StereoComponent/stereo_record.py

```diff
@@ -0,0 +1,32 @@
+import cv2
```

Comment:
do you think this script should go to `recording` folder instead?

Reply:
Moved it to another folder